web/src/test/java/com/graphhopper/http/GraphHopperServletIT.java

```diff
@@ -138,6 +140,23 @@ public void testGraphHopperWeb() throws Exception {
     }
 
     @Test
+    public void testNonTranslatedInstructions() {
+        GraphHopperAPI hopper = new GraphHopperWeb();
+        assertTrue(hopper.load(getTestRouteAPIUrl()));
+        GHResponse rsp = hopper.route(new GHRequest(42.554851, 1.536198, 42.510071, 1.548128));
+        assertEquals("Continue onto Carrer Antoni Fiter i Rossell", rsp.getBest().getInstructions().get(2).getName());
+
+        GHRequest request = new GHRequest(42.554851, 1.536198, 42.510071, 1.548128);
+        request.getHints().put("translate_instructions", false);
+        rsp = hopper.route(request);
+        assertEquals("Carrer Antoni Fiter i Rossell", rsp.getBest().getInstructions().get(2).getName());
```

Comment:
This is where it gets weird. If we use the raw name of the server side, this would mean the street name. getTurnDescription returns only the street name. On the client side using the raw name would mean the pre translated turn instruction that was returned from the server. So after thinking about this a bit more, how about `instructions_use_turn_description`?